src/Microsoft.Health.Dicom.Functions/Registration/ServiceCollectionExtensions.cs

```diff
@@ -53,11 +55,13 @@ public static class ServiceCollectionExtensions
         services.RegisterModule<ServiceModule>();
 
         return new DicomFunctionsBuilder(services
+            .AddInstanceCopier()
             .AddRecyclableMemoryStreamManager()
             .AddFellowOakDicomExtension()
             .AddFunctionsOptions<ExportOptions>(configuration, ExportOptions.SectionName)
             .AddFunctionsOptions<QueryTagIndexingOptions>(configuration, QueryTagIndexingOptions.SectionName, bindNonPublicProperties: true)
             .AddFunctionsOptions<PurgeHistoryOptions>(configuration, PurgeHistoryOptions.SectionName, isDicomFunction: false)
+            .AddFunctionsOptions<CopyOptions>(configuration, CopyOptions.SectionName, bindNonPublicProperties: true)
```

Comment:
Are you binding non-public properties?

Reply:
I dont think so, it came from copy paste